sdccc/src/main/java/com/draeger/medical/sdccc/tests/mdpws/invariant/InvariantSOAPOverHTTPTest.java

```diff
@@ -57,6 +57,9 @@ void testRequirement0006() throws IOException, NoTestData {
                     + " those circumstances.");
         }
 
+        // NOTE: the LocalizationService may also send very large messages. However, this test case
```

Comment:
I disagree with this comment, it could literally list all other services and doesn't add any benefit, e.g. GetService can trivially send very large messages as well.